pennylane/plugins/default_qubit.py

```diff
@@ -412,16 +414,24 @@ def sample(self, observable, wires, par):
     def _get_operator_matrix(self, operation, par):
         """Get the operator matrix for a given operation or observable.
 
+        If the inverse was defined for an operation or observable that requires parameters, returns the
+        operator matrix obtained using negated paramaters.
```

Comment:
This doesn't seem to match the logic:
```suggestion
        conjugate transpose of the operator matrix.
```